server/views/pages/appointments/create-and-edit/upload-prisoner-list.njk

```diff
@@ -2,12 +2,13 @@
 
 {% from "govuk/components/file-upload/macro.njk" import govukFileUpload %}
```

Comment:
It would be possible to add the changes from the upload-bulk-appointment.njk view into this one with checks for session.appointmentJourney.type == AppointmentType.BULK and then use this view in both routes. This is an optimisation so doesn't have to be done